modules/service/src/main/resources/db/migration/V0955__add_fluxDensitiesAttachment.sql

```diff
@@ -0,0 +1,3 @@
+-- Add fluxDensitiesAttachment to SEDs
+
+UPDATE t_target SET c_source_profile = REPLACE(c_source_profile::text, '"fluxDensities": null', '"fluxDensities": null, "fluxDensitiesAttachment": null')::jsonb;
```

Comment:
Hmm, if the decoder handles a missing `fluxDensitiesAttachment` maybe we don't need a migration after all?  If we do, would this fail for targets with a non-null `fluxDensities`?

Reply:
I don't think we have any targets with non-null fluxDensities, there's no way to do that from the UI. Someone would have had to call the API with that.

Reply:
I thought I could use `REGEXP_REPLACE` but it's not that simple since `fluxDensities` can contain an object. I suggest we leave it as it is, or remove the migration.

Reply:
Assuming it works to do so, and it seems like it should, I vote to remove the migration altogether.